addons/website/static/src/js/set_view_track.js

```diff
@@ -0,0 +1,74 @@
+odoo.define('website.set_view_track', function (require) {
+"use strict";
+
+var CustomizeMenu = require('website.customizeMenu');
+var rpc = require('web.rpc');
```

Comment:
If you have to require that, your code is very old ;) Use the `_rpc` from widgets to achieve the same result without this and allow testing this code properly some day.